apps/users/migrations/0002_text_to_htmlfield.py

```diff
@@ -1,24 +0,0 @@
-# Generated by Django 3.0.8 on 2020-09-22 07:22
-
-import djangocms_text_ckeditor.fields
-from django.db import migrations
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ("users", "0001_initial"),
-    ]
-
-    operations = [
-        migrations.AlterField(
-            model_name="board",
-            name="description",
-            field=djangocms_text_ckeditor.fields.HTMLField(blank=True),
-        ),
-        migrations.AlterField(
-            model_name="committee",
-            name="description",
-            field=djangocms_text_ckeditor.fields.HTMLField(blank=True),
-        ),
-    ]
```

Comment:
You cant just remove this migration, need to make sure that this migration is reversed in the database first, otherwise errors will occur.